pkg/setup-apply-firmware/main.go

```diff
@@ -40,9 +41,12 @@ func main() {
 	urls := os.Args[1:]
 	files := make([]string, len(urls))
 	for i, u := range urls {
-		idx := strings.LastIndex(u, "/")
-		f := u[idx+1:]
-		files[i] = path.Join(tmpdir, f)
+		url, err := url.Parse(u)
```

Comment:
`url` is already used for the package name.
This is not an error, but this is error-prone.

Reply:
Oops, I was typing it out of habit, so I didn't notice.
I'm surprised that this does not cause compile error.